engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/versions/views/content-view-version-yum.html

```diff
@@ -4,13 +4,26 @@
       <tr bst-table-head>
         <th bst-table-column translate>Name</th>
         <th bst-table-column translate>Product</th>
+        <th bst-table-column translate>Content</th>
       </tr>
     </thead>
 
     <tbody>
       <tr bst-table-row ng-repeat="repository in table.rows">
         <td bst-table-cell>{{ repository.name }}</td>
         <td bst-table-cell>{{ repository.product.name }}</td>
+        <td bst-table-cell>
+          <div>
+            <a ui-sref="products.repository.manage-content.packages({productId: repository.product.id, repositoryId: repository.id})">
```

Comment:
Sorry I missed this before but this needs to be translated via the translate directive.